handler.go

```diff
@@ -38,11 +39,21 @@ func newHandler(config []byte) (*handler, error) {
 		return nil, err
 	}
 	for _, e := range h.m {
-		if e.Display != "" {
-			continue
-		}
-		if strings.Contains(e.Repo, "github.com") {
+		switch {
+		case e.Display != "":
+			// Already filled in.
+		case strings.HasPrefix(e.Repo, "https://github.com/"):
 			e.Display = fmt.Sprintf("%v %v/tree/master{/dir} %v/blob/master{/dir}/{file}#L{line}", e.Repo, e.Repo, e.Repo)
+		case strings.HasPrefix(e.Repo, "https://bitbucket.org"):
+			e.Display = fmt.Sprintf("%v %v/src/default{/dir} %v/src/default{/dir}/{file}#{file}-{line}", e.Repo, e.Repo, e.Repo)
+		}
+		switch {
+		case e.VCS != "":
+			// Already filled in.
+		case strings.HasPrefix(e.Repo, "https://bitbucket.org/"):
+			e.VCS = "hg"
```

Comment:
In thinking about it, even assuming Mercurial isn't guaranteed to work out on Bitbucket &mdash; they also host Git now.  Seems like the safest behavior then is to make `newHandler` return a configuration error if we get into a case where we can't autodetect.

Does that seem too onerous for users?